src/internal/tracing/extended/extended_trace.go

```diff
@@ -100,7 +102,7 @@ func PersistAny(ctx context.Context, c *etcd.Client, pipeline string) {
 	}
 	if _, err := col.NewSTM(ctx, c, func(stm col.STM) error {
 		tracesCol := TracesCol(c).ReadWrite(stm)
-		return errors.EnsureStack(tracesCol.PutTTL(pipeline, traceProto, int64(duration.Seconds())))
+		return errors.EnsureStack(tracesCol.PutTTL(pipeline.String(), traceProto, int64(duration.Seconds())))
 	}); err != nil {
 		log.Errorf("could not persist extended trace for pipeline %q to etcd: %v",
 			pipeline, err)
```

Comment:
Do you want `pipeline.String()` here too?

Reply:
It’s called when using `%q`, and https://github.com/pachyderm/pachyderm/blob/3fc355fc4d979f42c77b1763247b227223ebe898/src/pps/pps.go#L11 will include the project name if any.